rasa/validator.py

```diff
@@ -26,27 +27,37 @@ class Validator:
     """A class used to verify usage of intents and utterances."""
 
     def __init__(
-        self, domain: Domain, intents: TrainingData, story_graph: StoryGraph
+        self,
+        domain: Domain,
+        intents: TrainingData,
+        story_graph: StoryGraph,
+        config: Optional[Dict[Text, Any]],
     ) -> None:
-        """Initializes the Validator object. """
+        """Initializes the Validator object.
 
+        Args:
+            domain: The domain.
+            intents: Training data.
+            story_graph: The story graph.
+            config: The configuration.
+        """
         self.domain = domain
         self.intents = intents
         self.story_graph = story_graph
+        self.nlu_config = RasaNLUModelConfig(config)
 
     @classmethod
     async def from_importer(cls, importer: TrainingDataImporter) -> "Validator":
         """Create an instance from the domain, nlu and story files."""
-
         domain = await importer.get_domain()
         story_graph = await importer.get_stories()
         intents = await importer.get_nlu_data()
+        config = await importer.get_config()
```

Comment:
where is the `config` coming from? I don't see a way to provide it in the CLI

Reply:
I'll add a `--config` parameter.